dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ClusterStats.java

```diff
@@ -0,0 +1,26 @@
+package com.dotcms.content.elasticsearch.business;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClusterStats {
+
+    private String clusterName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'clusterName' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)